src/components/CardListBox/CardFolderList/CardFolderList.tsx

```diff
@@ -0,0 +1,33 @@
+import Folder from "./Folder/Folder";
+import styles from "./CardFolderList.module.scss";
+import { UserFolderProps } from "@/constants/index.types";
+
+interface CardFolderListProps {
+  folders?: UserFolderProps[];
+  handleFolderClick: (folder: UserFolderProps) => void;
+  handleTotalBtnClick: () => void;
```

Comment:
개인적으로는 축약하지 않고 작성이 되면 더 좋을 것 같아요~
```suggestion
  handleFolderClick: (folder: UserFolderProps) => void;
  handleTotalButtonClick: () => void;
```